packages/st2/debian/st2.st2api.init

```diff
@@ -71,6 +71,8 @@ do_start()
   start-stop-daemon --start --quiet --chuid $RUNAS_USER:$RUNAS_GROUP --umask $UMASK --background --pidfile $PIDFILE \
      --exec $DAEMON -- $DAEMON_ARGS \
     || return 2
+  # wait for st2api readiness to serve the requests
+  timeout 5 bash -c 'until echo > /dev/tcp/127.0.0.1/9101; do sleep 0.5; done' || return 2
```

Comment:
I'm just not sure about `5s` timeout, looks a bit high value.
Probably retries during the 3s timeout should be more than enough to understand if there is something wrong with the service and it's broken.